#160886981 Create /auth/signup , /auth/login endpoints

sekayasin opened the pull request:
## What does this PR do?
Have signup and login for the fast-food-fast API

### Description of the task to be completed
POST /auth/signup
POST /auth/login
With those endpoints, a user can successfully create an account on fast-food-fast, the data is successfully stored in fast-food-fast db.

POST /auth/login - This endpoint at moment do tests with a username=test and password=test and generate an access token with jwt successfully.

sekayasin committed:
ft-signin-signup

Add Database, Register users and store to db.
Add /auth/signup, /auth/login endpoints

[Delivers #160886981]

Kasulejoseph commented:
- And always don't forget to add a screenshot of your postman
- Add a story id as well

sekayasin commented:
ooh yes, @Kasulejoseph thanks

Kasulejoseph commented on api/routes.py:
```diff
@@ -24,73 +33,77 @@ def index():
+    access_token = create_access_token(identity=username)
+    return jsonify(access_token=access_token), status.HTTP_200_OK
+
+# Protect a view with jwt_required, which requires a valid access token
```
In-line comments is also not recommended in PEP8 so avoid it

sekayasin replied:
Yes

sekayasin committed:
ft-signin-signup

Modified /auth/login to check username and password during login

[Delivers #160886981]
